lib/components/Button/Button.tsx

```diff
@@ -7,20 +7,11 @@ export interface ButtonProp extends React.ComponentPropsWithoutRef<'button'> {
   isSmall?: boolean
   isFull?: boolean
   isDisabled?: boolean
-}
-
-export interface ButtonTitleProps extends ButtonProp {
-  title: string
+  title?: string
   icon?: React.ForwardRefExoticComponent<Omit<React.SVGProps<SVGSVGElement>, 'ref'>>
   ariaLabel?: string
 }
 
-export interface ButtonNoTitleProps extends ButtonProp {
```

Comment:
@troychaplin I would like these 2 conditional props. if there is not title then aria-label must be there

Reply:
Makes sense, revert back to multiple interfaces